src/Calculator/Views/CalculatorProgrammerRadixOperators.xaml.cpp

```diff
@@ -176,3 +177,23 @@ void CalculatorProgrammerRadixOperators::ClearButton_LostFocus(Object ^ sender,
         ClearEntryButton->Focus(::FocusState::Programmatic);
     }
 }
+
+void CalculatorApp::CalculatorProgrammerRadixOperators::BitShiftFlyout_Opened(Platform::Object ^ sender, Platform::Object ^ e)
+{
+    if (m_selectedShiftButtonContent == m_arithmeticShiftButtonContent)
```

Comment:
Its by design that arithmetic shift is selected when switching away & back from other modes, this no longer happens with the graphing mode, but does with other modes, not sure why this occurs but it seems related to the root cause of the issue here.

Reply:
I'll roll back the checkDefaultBitShift function I deleted.

Reply:
I still see the same issue after your change.

To be more specific here is what I am seeing:
1. Go to programmer mode, change bit shift radio button to be logical
2. Go to graphing mode
3. Go back to programmer mode, open the bit shift flyout and see that the radio button is still on logical.

Repeat the above steps with any other mode in Calculator and you will see that the bit shift flyout will reset back to arithmetic shift.

Reply:
I understand what you say, but, in my opinion, this issue is not about my first issue.
I think this problem is about calling callback in Calculator ViewModel.
I'll comment more detail in related commit

Reply:
I just saw this before realizing it myself, you are right it is a pre-existing problem.